Documents/Documents/Code/Controllers/Documents/Menu/CurrentFolderMenu/CurrentRoomArchivesMenu.swift

```diff
@@ -12,9 +12,6 @@ final class CurrentRoomArchivesMenu: CurrentFolderMenuProtocol {
     private lazy var sortTypes: [ASCDocumentSortType] = [.az, .type, .tag, .author, .dateandtime]
 
     func contextMenu(for folder: ASCFolder, in viewController: ASCDocumentsViewController) -> UIMenu {
-        guard let provider = viewController.provider else { return UIMenu() }
-        let actions = provider.actions(for: folder)
```

Comment:
It'll be better to get correct set off actions here form provider instead removing items here

Reply:
The logic for displaying the action menu in the archive room differs from the standard one. In the action menu of the archive room, only the filter and select button are displayed.